File: TrackMyAct.Server/Models/Entities/UserEntity.cs
namespace TrackMyAct.Server.Models.Entities;

public class UserEntity : Entity
{
    public string Username { get; set; }
    public string Email { get; set; }
    public string PasswordHash { get; set; }
    public string Name { get; set; }
    public string Phone { get; set; }
    public DateTime BirthDate { get; set; }
}

File: TrackMyAct.Mobile/composeApp/src/commonMain/kotlin/org/js/tma/dto/UserEntity.kt
package org.js.tma.dto

import kotlinx.serialization.Serializable

@Serializable
abstract class UserEntity : Entity() {
    abstract var username: String
    abstract var email: String
    abstract var passwordHash: String
    abstract var name: String
    abstract var roleType: String // Participant/Organizer
}